src/app/containers/ProfilePage/index.js

```diff
@@ -61,47 +72,44 @@ function ProfilePage({ match }) {
               <div className={styles.ava}>
                 <img src={img} alt="avatar" />
               </div>
-              <div className={styles.changeAvatar}>
-                {currentUsername === match.params.username && (
-                  <Link to="/settings">Thay đổi ảnh cá nhân</Link>
-                )}
-              </div>
             </div>
             <div className={styles.infoContent}>
               <div>
-                <h3>{userInfo.username}</h3>
-              </div>
-              <div>
-                <span>Họ và tên:</span> {userInfo.fullName}
-              </div>
-              <div>
-                <span>Ngày sinh:</span> {userInfo.dateOfBirth}
-              </div>
-              <div>
-                <span>Trường học:</span> {userInfo.schoolName}
+                <h3>{handlingUserInfo.username} </h3>
+                {userInfo.username === match.params.username && (
+                  <Link to="/settings">
```

Comment:
using constant `ROUTE_SETTING` from `src/app/routes/constants`. So when we modify in the constants file, this file will be also updated